src/drf_yasg/generators.py

```diff
@@ -89,8 +102,12 @@ def get_api_endpoints(self, patterns=None, prefix='', app_name=None, namespace=N
                 try:
                     path = self.get_path_from_regex(path_regex)
                     callback = pattern.callback
+                    versioning_class = getattr(callback.cls, 'versioning_class', None)
+                    version = getattr(self.request, 'version', None)
+                    version_param = version and getattr(versioning_class, 'version_param', 'version')
+                    version_regex = self.get_version_regex_from_path_regex(path_regex, version_param)
```

Comment:
Why are this lines here and not in `should_include_endpoint`?

Reply:
Thank you for pointing! Fixed.